source/Halibut.TestUtils.CompatBinary.Base/Halibut.TestUtils.CompatBinary.Base.csproj

```diff
@@ -11,6 +11,9 @@
     <PropertyGroup Condition="$([MSBuild]::IsOSUnixLike())">
         <TargetFramework>net6.0</TargetFramework>
     </PropertyGroup>
+    <ItemGroup>
+      <Compile Include="..\Halibut.Tests\Support\TcpPortHelper.cs" Link="TcpPortHelper.cs" />
+    </ItemGroup>
```

Comment:
...for when you absolutely do not want to have to create a new project with a single class in it! 😛